sdk/attestation/azure-security-attestation/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "sourceFilesToProcess": [ "azure\\attestation.hpp" ],
```

Comment:
This relies on all public headers to be accessible from the root "simplified" header. Are wildcards allowed in this? Can we have all headers in the `inc` folder be processed, instead?

Reply:
This incantation is correct for attestation - all the headers in the tree are referenced by attestation.hpp.

It's not correct for other components (like Azure::Core), which is why they don't specify a `sourceFilesToProcess` directive.

Reply:
@ahsonkhan, @LarryOsterman, would it be helpful if we implement #4234?

Reply:
That's going to be a challenge for core - there are a number of _internal only headers which should not be present in the uber header. The same is likely true for Storage::Common.